flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormat.java

```diff
@@ -143,6 +144,11 @@ public void openInputFormat() {
 			} else {
 				dbConn = DriverManager.getConnection(dbURL, username, password);
 			}
+
+			if (autoCommit != null) {
```

Comment:
Add a comment like: "Override dafault auto commit mode only if it was explicitly set."?